speakers.js

```diff
@@ -0,0 +1,90 @@
+const speakersSection = document.querySelector('#speakers');
+
+const speakersObject = [
+  {
+    name: 'H.E Hakainde Hichilema',
+    about: 'Current president of Zambia',
+    history:
+      'He has been in opposition for 20 years failed 4 times. he owns shares in big global companies.',
```

Comment:
- [ ] Please your object should contains all the information needed to generate the dynamic element, at least the name, role, description, and image url about the speaker. 
````
{
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
}
````

Reply:
- [ ] [REQUIRED] As recommended by the previous reviewer, kindly ensure that the object contains all the information needed to generate the dynamic element.  Please remove the images and populate them using Javascript.

```
{
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
}
```